<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scenario_view_widget</class>
 <widget class="QWidget" name="scenario_view_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1617</width>
    <height>979</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,3,0">
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Scenario Viewer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="scenario_help">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../resources.qrc">
             <normaloff>:/icon/images/icons/ainfo_24dp_5F6368_FILL0_wght200_GRAD0_opsz24.png</normaloff>:/icon/images/icons/ainfo_24dp_5F6368_FILL0_wght200_GRAD0_opsz24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="styleSheet">
           <string notr="true">
	background-color: rgb(208, 208, 208);
	border-radius: 25px;

	color: black;


</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_excel">
             <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
              <widget class="QPushButton" name="load_button">
               <property name="styleSheet">
                <string notr="true">QPushButton {
        border: 1px solid rgb(40, 84, 113);
        border-radius: 7px;
        background-color: rgb(40, 84, 113);
        color: white;
		 font: 700 10pt &quot;Segoe UI&quot;;
        padding: 1px;
    }
QPushButton:hover {
        background-color: rgb(60, 120, 150);
    }
QPushButton:pressed {
        background-color: rgb(20, 60, 80);
    }</string>
               </property>
               <property name="text">
                <string>Load Excel Files</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="styleSheet">
           <string notr="true">
	background-color: rgb(208, 208, 208);
	border-radius: 25px;

	color: black;


</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_select">
             <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Select Scenarios</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="styleSheet">
           <string notr="true">
	background-color: rgb(208, 208, 208);
	border-radius: 25px;

	color: black;


</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_selected">
             <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Selected</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="graph_type_combo">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255,255,255);</string>
            </property>
            <item>
             <property name="text">
              <string>Generation Capacity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ESS Capacity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Capacity Investments</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="plot_button">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
        border: 1px solid rgb(40, 84, 113);
        border-radius: 7px;
        background-color: rgb(40, 84, 113);
        color: white;
		 font: 700 10pt &quot;Segoe UI&quot;;
        padding: 1px;
    }
QPushButton:hover {
        background-color: rgb(60, 120, 150);
    }
QPushButton:pressed {
        background-color: rgb(20, 60, 80);
    }</string>
            </property>
            <property name="text">
             <string>Plot</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progbar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QProgressBar{
	border: 1px solid#cfcfcf;
	border-radius:8px;
	background-color: transparent;
	text-align: center;
	font: bold 10pt &quot;Segoe UI&quot;;
	color: #444;
	height:16px;
}
QProgressBar::chunk {
	border-radius:8px;
	background:qlineargradient(
	spread:pad, x1:0, y1:0, x2:1, y2:0,
	stop:0 #6aa9ff,
	stop:1 #005bea
);
}</string>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
